carla/recourse_methods/autoencoder/models.py

```diff
@@ -306,19 +307,19 @@ def fit(
 
             ELBO[epoch] = train_loss / train_loss_num
             if epoch % 10 == 0:
-                print(
+                log.info(
```

Comment:
Are these logs really logs a user of this library want to see on every run or is this something that could rather be a `log.debug()`? And if yes should we us a bit more context? 

Like this its hardly recognizable that the epocs are for the autoencoder but could also be any epochs of training anything

Reply:
Hmm I think the training of an AE would be important information.
But it would be way better mentioning the training before 😄 Will change this!